public/data/Ayush-Kanduri.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "Ayush Kanduri",
+  "type": "personal",
+  "bio": "Fullstack Web Developer (MERN) 🔥 | Exploring Open Source ⭐ | Tech Enthusiast 🚀",
+  "avatar": "https://github.com/Ayush-Kanduri.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Ayush-Kanduri",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Ayush_Codes",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ayushkanduri",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Subscribe to my channel on YouTube",
+      "url": "https://www.youtube.com/user/MrAyushChamp",
+      "icon": "youtube"
+    },
+    {
+      "name": "Checkout my Portfolio",
+      "url": "https://ayushkanduri.dev",
```

Comment:
There is a broken link, please update it with the correct one! 

![image](https://user-images.githubusercontent.com/88786642/193978561-1b44dadc-7588-42b3-aec6-b597dab344d0.png)

Reply:
Hi Priyanshu. Resolved the error. Kindly do the needful.